src/test/java/com/easypost/OrderTest.java

```diff
@@ -92,4 +94,33 @@ public void testBuy() throws EasyPostException {
             assertNotNull(shipment.getPostageLabel());
         }
     }
+
+    /**
+     * Test getting lowest rate of an Order.
+     *
+     * @throws EasyPostException when the request fails.
+     */
+    @Test
+    public void testLowestRate() throws EasyPostException {
+        Order order = Order.create(Fixture.basicOrder());
+
+        // Test lowest rate with no filters
+        Rate lowestRate = order.lowestRate();
+        assertEquals("First", lowestRate.getService());
+        assertEquals(5.49, lowestRate.getRate(), 0.01);
+        assertEquals("USPS", lowestRate.getCarrier());
+
+        // Test lowest rate with service filter (this rate is higher than the lowest but should filter)
+        List<String> service = new ArrayList<>(Arrays.asList("Priority"));
+        Rate lowestRateService = order.lowestRate(null, service);
+        assertEquals("Priority", lowestRateService.getService());
+        assertEquals(7.37, lowestRateService.getRate(), 0.01);
+        assertEquals("USPS", lowestRateService.getCarrier());
+
+        // Test lowest rate with carrier filter (should error due to bad carrier)
+        List<String> carrier = new ArrayList<>(Arrays.asList("bad carrier"));
```

Comment:
Please use uppercase to call to the reader's mind it's out of place
```suggestion
        List<String> carrier = new ArrayList<>(Arrays.asList("BAD CARRIER"));
```